Rakefile

```diff
@@ -6,11 +6,24 @@ require 'rake/extensiontask'
 require_relative './ext/tiny_tds/extconsts'
 
 SPEC = Gem::Specification.load(File.expand_path('../tiny_tds.gemspec', __FILE__))
+
+ruby_cc_ucrt_versions = "3.1.0".freeze
+ruby_cc_mingw32_versions = "3.0.0:2.7.0:2.6.0:2.5.0:2.4.0".freeze
+
 GEM_PLATFORM_HOSTS = {
-  'x86-mingw32' => 'i686-w64-mingw32',
-  'x64-mingw32' => 'x86_64-w64-mingw32'
+  'x64-mingw32' => {
+    host: 'x86_64-w64-mingw32',
+    ruby_versions: ruby_cc_mingw32_versions
+  },
+  'x64-mingw-ucrt' => {
```

Comment:
Looks like this commit is also introducing a ucrt based build, can you update the commit message with an additional explanation on this?

Reply:
I added a reference to the commit in [cae55bf](https://github.com/rails-sqlserver/tiny_tds/pull/533/commits/cae55bf54932f63be4175dab9b3137acb3dbb037).